.gitignore

```diff
@@ -58,10 +60,16 @@ vendor/luajit/src/luajit
 .vs
 
 # Meson
+builddir/
```

Comment:
I had intentionally not included this in the interest of not having a fixed build directory name, but maybe it's fine. I had hoped the directory could just be called 'build' if we were going to have a fixed one, but I guess that's still not possible due to all the MSVC crap still in there. I think fine with this, but it may change down the line once I can get rid of that directory completely.